sdk/selfhelp/Azure.ResourceManager.SelfHelp/src/Generated/SelfHelpDiagnosticData.cs

```diff
@@ -52,10 +52,10 @@ public SelfHelpDiagnosticResourceData()
         /// Array of Diagnostics.
         /// Serialized Name: DiagnosticResource.properties.diagnostics
         /// </param>
-        internal SelfHelpDiagnosticResourceData(ResourceIdentifier id, string name, ResourceType resourceType, SystemData systemData, IDictionary<string, string> globalParameters, IList<DiagnosticInvocation> insights, string acceptedTime, ProvisioningState? provisioningState, IReadOnlyList<SelfHelpDiagnostic> diagnostics) : base(id, name, resourceType, systemData)
```

Comment:
Is `acceptedTime` really a string? Could it be a `DateTimeOffset`? See an example [here](https://github.com/Azure/azure-sdk-for-net/blob/e4fce1607d65d709b19c81cc78969cd08b804244/sdk/hdinsight/Azure.ResourceManager.HDInsight/src/autorest.md?plain=1#L146).

Reply:
This was a decision taken a while back to keep it string. We will be fixing it in the future versions.